assets/style/services/services.scss

```
@@ -5,3 +5,329 @@
+  }
+  @media (max-width: 480px) {
+    display: none;
+  }
+}
+.main__drilling-work-inform-bt {
+  @media (max-width: 1400px) {
+    text-align: center;
+    padding-bottom: 40px;
+  }
+  @media (max-width: 480px) {
+    padding-bottom: 15px;
+  }
+  @media (max-width: 360px) {
+    padding-bottom: 5px;
+  }
+  .main__drilling-work-inform-bt-btn {
+    @include default-btn(237px, $text-light-color, $btn-link-dark-color, 0px);
+    @media (max-width: 480px) {
+      padding-block: 8px 8px;
+      max-width: 200px;
+      line-height: 20px;
+    }
+    @media (max-width: 360px) {
+      font-size: small;
+    }
+  }
+}
+.main__map-box {
+  @media (max-width: 480px) {
+    display: none;
+  }
+  .main__map {
+    position: relative;
+  }
+  .main__map-contacts {
+    margin-inline: auto;
+    background-color: $text-light-color;
+    font-family: $font;
+    max-width: 650px;
+    padding: 40px;
+    position: absolute;
+    top: 115px;
+    left: 160px;
+    z-index: 2;
+    @media (max-width: 1900px) {
+      max-width: 33%;
+      top: 100px;
+      left: 140px;
+    }
+    @media (max-width: 1400px) {
+      padding: 30px;
+      top: 25px;
+      left: 25px;
+    }
+    @media (max-width: 1024px) {
+      max-width: 50%;
+      top: 65px;
+    }
+    @media (max-width: 640px) {
+      max-width: 70%;
+    }
+    .main__map-contacts-all {
+      padding-bottom: 14px;
+      @media (max-width: 1024px) {
+        display: none;
+      }
+      .main__map-contacts-questions {
+        font-weight: 700;
+        font-size: 45px;
+        line-height: 56px;
+        color: $text-dark-color;
+        @media (max-width: 1400px) {
+          font-size: 35px;
+          line-height: 40px;
+        }
+      }
+      .main__map-contacts-email {
+        font-weight: 700;
+        font-size: 35px;
+        line-height: 46px;
+        color: $btn-link-dark-color;
+        @media (max-width: 1400px) {
+          font-size: 30px;
+        }
+      }
+    }
+    .main__map-contacts-all-info {
+      display: grid;
+      grid-template-columns: 300px 3fr;
+      grid-template-rows: 100px 100px 50px;
+      gap: 15px;
+      @media (max-width: 1900px) {
+        grid-template-columns: 50% 50%;
+      }
+      @media (max-width: 1400px) {
+        grid-template-columns: none;
+        grid-template-rows: none;
+      }
+      .main__map-contacts1,
+      .main__map-contacts2 {
+        width: 300px;
+        display: flex;
+      }
+      .main__map-contactsh2 {
+        font-weight: 700;
+        font-size: 20px;
+        color: $text-dark-color;
+        padding-bottom: 15px;
+      }
+      p {
+        @include default-p($text-blue-color);
+        color: #949393;
+      }
+      .main__map-contactsh3 {
+        font-weight: 700;
+        font-size: 20px;
+        color: $text-dark-color;
+        padding-bottom: 15px;
+      }
+    }
+    .main__map-contacts1-img,
+    .main__map-contacts2-img {
+      .img {
+        padding-right: 10px;
```

Тут лучше использовать марджины, из-за того же нюанса с возможным цветовым фоном, о котором я говорил выше.
```suggestion
        margin-right: 10px;
```